docs/markdown/Using Pants/concepts/goals.md

```diff
@@ -38,8 +38,7 @@ You can also run multiple goals in a single run of Pants, in which case they wil
 ❯ pants fmt test ::
 ```
 
-Finally, Pants supports running goals in a `--loop`. In this mode, all goals specified will run  
-sequentially, and then Pants will wait until a relevant file has changed to try running them again.
+Finally, Pants supports running goals in a `--loop`. In this mode, all goals specified will run sequentially, and then Pants will wait until a relevant file has changed to try running them again.
```

Comment:
Simply removed the linebreak as readme.com doesn't support it; no changes to the content.